src/ContributorLicenseAgreement.Core/Handlers/IssueCommentHandler.cs

```diff
@@ -174,17 +182,19 @@ private async Task<bool> CheckSenderAsync(GitOpsPayload gitOpsPayload)
             }
         }
 
-        private (CommentAction, string) ParseComment(string comment, string host, Cla primitive)
+        private (CommentAction, string) ParseComment(
+            string comment,
+            string gitHubAppName,
+            Cla primitive)
         {
-            var bot = $"@{flavorSettings[host].Name}";
-            var cleanedComment = ExtractPolicyServiceLine(comment, bot);
+            var cleanedComment = ExtractPolicyServiceLine(comment, gitHubAppName);
             var pattern = @"[ ](?=(?:[^""]*""[^""]*"")*[^""]*$)";
             var regex = new Regex(pattern);
             var tokens = regex.Split(cleanedComment);
 
             CommentAction commentAction = CommentAction.Failure;
 
-            if (tokens.Length >= 2 && tokens.First().StartsWith(bot))
+            if (tokens.Length >= 2 && tokens.First().StartsWith(gitHubAppName))
```

Comment:
if (tokens.Length >= 2 && tokens.First().StartsWith($"@{gitHubAppName}"))
this would be the correct line